axonserver/src/main/java/io/axoniq/axonserver/metric/ClusterMetric.java

```diff
@@ -15,13 +15,13 @@
  */
 public interface ClusterMetric {
 
-    long size();
+    long value();
 
     long min();
 
     long max();
 
     double mean();
 
-
+    double doubleValue();
```

Comment:
if the reason to add doubleValue method is that the metric can be represented by a double, why not to generify the number type instead of adding a method?